.github/workflows/linux-raspbian.yml

```diff
@@ -0,0 +1,140 @@
+name: Ubuntu
+
+on: [push, pull_request]
+
+env:
+  APPIMAGEVER: 13
+  PPAQTVER: 515
+  QTIFWVER: 4.5.1
+  QTVER: 5.15.2
+  NJOBS: 4
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 10
+      fail-fast: false
+      matrix:
+       include:
+         - dockerimg: ubuntu:focal
+           compiler: gcc
+           upload: 1
+         - dockerimg: ubuntu:focal
+           compiler: clang
+           upload: 0
+         - dockerimg: ubuntu:rolling
+           compiler: gcc
+           upload: 0
+         - dockerimg: ubuntu:rolling
+           compiler: clang
+           upload: 0
+#         - dockerimg: ubuntu:devel
+#           compiler: gcc
+#           upload: 0
+#         - dockerimg: ubuntu:devel
+#           compiler: clang
+#           upload: 0
+    steps:
+    - uses: actions/checkout@v2
+    - uses: pguyot/arm-runner-action@v2
+      id: build_image
+      with:
+        base_image: raspios_lite:2022-04-04
```

Comment:
I was thinking, instead of using this image, I'm more in the side of using one of the [official images](https://hub.docker.com/search/?q=&type=image&image_filter=official&operating_system=linux&architecture=arm%2Carm64), and [Ubuntu](https://hub.docker.com/_/ubuntu) already provide images for ARM and ARM64, and in the same was as the official Webcamoid releases are built using an Ubuntu image, and it works in Arch, Fedora, Debian, etc., if we build the binaries in Ubuntu ARM it should be able to work in Raspbian, Arch Linux ARM, or any other ARM based distro.
So the code above is not really necessary but instead you must configure docker to point to the ARM and ARM64 images, look at [here](https://hub.docker.com/r/arm64v8/ubuntu/) for example (at **What's in this image?**), apparently images are indicated as ```ARCHITECTURE/ubuntu:latest```.